### ISSUE TITLE
Elm 0.19 support

### ISSUE THREAD

**Issue by arsduo** [arsduo/elm-rings#4]
Elm 0.19 changes some of the details ElmRings relies on; the library needs to be upgraded.

This will happen once Webpack support is fixed for the new version so that I can upgrade and test against the example (see https://github.com/elm-community/elm-webpack-loader/pull/142).

**Comment by arsduo** [comment]
The Webpack loader works, but now I'm blocked by another issue I've found and reported to the Elm team: https://github.com/elm/compiler/issues/1799
